coalib/bears/Bear.py

```diff
@@ -249,8 +250,19 @@ def execute(self, *args, **kwargs):
             result = self.run_bear_from_section(args, kwargs)
             return [] if result is None else list(result)
         except (Exception, SystemExit):
-            self.warn('Bear {} failed to run. Take a look at debug messages'
-                      ' (`-V`) for further information.'.format(name))
+            try:
+                if self.kind() == BEAR_KIND.LOCAL:
+                    self.warn('Bear {} failed to run on file {}. Take a look '
+                              'at debug messages (`-V`) for further '
+                              'information.'.format(name, args[0]))
```

Comment:
I think we should keep 2 separate messages for better readability IMO @Makman2 @Asnelchristian